src/sites/twitch-twilight/modules/host-options.vue

```diff
@@ -0,0 +1,176 @@
+							<div class="ffz--widget ffz--checkbox">
+								<div class="flex align-items-center">
+									<input type="checkbox" class="tw-checkbox__input"
+										id="autoHostSettings:enabled"
+										data-setting="enabled"
+										:checked="autoHostSettings.enabled"
+										@change="updateCheckbox">
+									<label for="autoHostSettings:enabled" class="tw-checkbox__label">
+										Auto Hosting
+									</label>
+								</div>
+								<section class="c-text-alt-2" style="padding-left: 2.2rem;">
+									Toggle all forms of auto hosting: teammates, host list, and similar channels.<br>
+									<a href="https://blog.twitch.tv/grow-your-community-with-auto-hosting-e80c1460f6e1" target="_blank" rel="noopener">Learn More</a>
+								</section>
+							</div>
+							<div class="ffz--widget ffz--checkbox">
+								<div class="flex align-items-center">
+									<input type="checkbox" class="tw-checkbox__input"
+										id="autoHostSettings:team_host"
+										data-setting="team_host"
+										:checked="autoHostSettings.team_host"
+										@change="updateCheckbox">
+									<label for="autoHostSettings:team_host" class="tw-checkbox__label">
+										Team Hosting
+									</label>
+								</div>
+								<section class="c-text-alt-2" style="padding-left: 2.2rem;">
+									Automatically host random channels from your team when you're not live.<br>
+									Team channels will be hosted before any channels in your host list.
+								</section>
+							</div>
+							<div class="ffz--widget ffz--checkbox">
+								<div class="flex align-items-center">
+									<input type="checkbox" class="tw-checkbox__input"
+										id="autoHostSettings:vodcast_hosting"
+										data-setting="deprioritize_vodcast"
+										:checked="!autoHostSettings.deprioritize_vodcast"
+										@change="updateCheckbox">
+									<label for="autoHostSettings:vodcast_hosting" class="tw-checkbox__label">
+										Vodcast Hosting
+									</label>
+								</div>
+								<section class="c-text-alt-2" style="padding-left: 2.2rem;">
+									Include Vodcasts in auto host.<br>
+									<a href="https://blog.twitch.tv/vodcast-brings-the-twitch-community-experience-to-uploads-54098498715" target="_blank" rel="noopener">Learn about Vodcasts</a>
+								</section>
+							</div>
+							<div class="ffz--widget ffz--checkbox">
+								<div class="flex align-items-center">
+									<input type="checkbox" class="tw-checkbox__input"
+										id="autoHostSettings:recommended_host"
+										data-setting="recommended_host"
+										:checked="autoHostSettings.recommended_host"
+										@change="updateCheckbox">
+									<label for="autoHostSettings:recommended_host" class="tw-checkbox__label">
+										Auto-Host Channels Similar To Yours
+									</label>
+								</div>
+								<section class="c-text-alt-2" style="padding-left: 2.2rem;">
+									Streamers on your primary team & host list will always be hosted first
```

Comment:
Needs translation support. Should be using an HTML entity and not a raw ampersand, probably.